scripts/src/primazactl/primaza/primaza.py

```diff
@@ -0,0 +1,81 @@
+import yaml
+from typing import Dict
+from primazactl.utils import logger, command
+from primazactl.identity import kubeidentity
+from primazactl.kube.secret import Secret
+from primazactl.utils import kubeconfig
+from primazactl.utils.kubeconfigwrapper import KubeConfigWrapper
+
+
+class Primaza(object):
```

Comment:
may `Cluster` or `PrimazaCluster` be a more appropriate name?

```suggestion
class PrimazaCluster(object):
```

Reply:
Sure, changed to that.